### PR TITLE
Added "task 2 - promt" description

### PR DESCRIPTION
**Pull request by gologames**

### CODE REVIEW

**Comment by gologames** [comment]
@AnastasiaMamontova please check the description of new task 2. Approve the request if you understand the task

**Comment by AnastasiaMamontova** [comment]
write prompt programm to ask about cost

**Review by AnastasiaMamontova — COMMENTED** [review]
prompt

**Review by AnastasiaMamontova — APPROVED** [review]

File: Tasks/2 - Prompt/promt.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+	<head></head>
+	<body>
+		<script>
+		let cost = parseFloat(prompt("How much does a loaf of bread cost?"));
```
Comment (gologames) [inline Tasks/2 - Prompt/promt.html]: Please change every `let` to `const` due to program doesn't modify these variables

File: Tasks/2 - Prompt/promt.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+		let number = parseFloat(prompt("How many loaves do you want?"));
+		let result = cost * number;
+		alert ("The total cost is $" + " " + result);
```
Comment (gologames) [inline Tasks/2 - Prompt/promt.html]: Remove unnessesary space symbol after `alert`

Also try to rewrite expression `"The total cost is $" + " " + result` with [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) technique

File: Tasks/2 - Prompt/promt.html (hunk 1 above)
Comment (AnastasiaMamontova) [reply Tasks/2 - Prompt/promt.html]: have done

File: Tasks/2 - Prompt/promt.html (hunk 2 above)
Comment (AnastasiaMamontova) [reply Tasks/2 - Prompt/promt.html]: have done

**Pull request merged**